Rubberduck.VBEEditor/ComManagement/TypeLibs/TypeLibsAPI.cs

```diff
@@ -35,6 +45,13 @@ public static bool CompileProject(IVBE ide, string projectName)
                 return typeLibs.FindTypeLib(projectName).CompileProject();
             }
         }
+        public static bool CompileProject(IVBProject project)
```

Comment:
Typically, a way to minimize the code duplication when overloading is to have them call the most-overloaded method.

Example:
```
public void Foo(string Bar)
{
  Foo(GetObjectByName(Bar))
}

public void Foo(SomeObject Bar)
{
  // actual code...
}
```

I would refactor those so that the those taking `string` calls the `IVBProject` overload, leaving us with only one implementation.

Reply:
yup, I'll sort that